src/main/java/io/zeebe/Application.java

```diff
@@ -9,17 +9,45 @@
 import java.util.Map;
 import java.util.Scanner;
 
+/**
+ * Sample application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
```

Comment:
```suggestion
 * Example application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
```

I prefer the word example over sample. I feel like the word example means an illustration or a case while a sample is more a specimen. But please ignore this if you feel differently.